api/src/test/java/jakarta/data/spi/expression/literal/LiteralRecordTest.java

```diff
@@ -63,16 +61,12 @@ void shouldNotEqualDifferentValues() {
         var one = new LiteralRecord<>("one");
         var two = new LiteralRecord<>("two");
 
-        SoftAssertions.assertSoftly(soft -> {
-            soft.assertThat(one).isNotEqualTo(two);
-        });
+        SoftAssertions.assertSoftly(soft -> soft.assertThat(one).isNotEqualTo(two));
     }
 
     @Test
     @DisplayName("should throw NullPointerException when value is null")
     void shouldThrowWhenValueIsNull() {
-        org.junit.jupiter.api.Assertions.assertThrows(NullPointerException.class, () -> {
-            new LiteralRecord<>(null);
-        }, "The value argument is required");
+        org.junit.jupiter.api.Assertions.assertThrows(NullPointerException.class, () -> new LiteralRecord<>(null), "The value argument is required");
```

Comment:
Another example of where the PR makes formatting changes that condense code onto really long lines (in this case 150 characters in a single line) contrary to the agreed upon line length limit.

```suggestion
        org.junit.jupiter.api.Assertions.assertThrows(
                NullPointerException.class,
                () -> new LiteralRecord<>(null),
                "The value argument is required");
```